tox.ini

```
@@ -2,14 +2,12 @@
 max-line-length = 99
 
 [tox]
-envlist = py27,flake8
+envlist = py36, py37, flake8
```

We should take this time to specify the version of Python used with flake8. Can just be this: 
```suggestion
envlist = py36, py37, py37-flake8
```

---

You would also need to change the testenv name on line 12, and also in the Travis config to define the version of Python and use the new testenv name.